changelog/_unreleased/2020-09-17-next-10854-support-custom-cache-folders.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
No need to mention the ticket in the changelog filename

Reply:
file renamed, will remind that for the next pr